backend/src/main/java/zipgo/review/application/ReviewService.java

```diff
@@ -31,4 +32,21 @@ public Long createReview(Long memberId, CreateReviewRequest request) {
         return review.getId();
     }
 
+    public void updateReview(Long memberId, Long reviewId, UpdateReviewRequest request) {
+        Review review = reviewRepository.getById(reviewId);
+
+        update(memberId, request, review);
```

Comment:
```suggestion
        Review review = reviewRepository.getById(reviewId);
        review.validateOwner(memberId);
        update(request, review); //업데이트만 수행
```
제 짐작으로는, Review 안의 여러 항목들을 업데이트하는 로직을 `리뷰 업데이트`로 추상화하기 위해 메서드를 분리한 것 같은데 맞을까요?! 클만약 그렇다고 하면 검증부는 포함하지 않는 것도 좋을 듯 해요.

무민 생각도 동일하다면~ 반영해주시면 될 거같고 아님 생각공유해주시고 넘어가도 좋습니다!